LICENSE.txt

```diff
@@ -609,4 +609,10 @@ Copyright 2022 Apache Lucene.NET
 
 Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the License for the specific language governing permissions and limitations under the License.
 
+Microsoft.Extensions.Configuration.Json v6.0.0
+The MIT License (MIT)
+https://github.com/dotnet/runtime/blob/main/LICENSE.TXT
```

Comment:
I am not sure how we handle this - @pinzart90 do you know? I don't think it hurts to list the nuget packages we reference, even if they are covered by the dotnet license - but it could be a lot of them all with the same license.

Reply:
if we decide to include this please follow the pattern and include the license text.